integration-tests/tests/analysis-context-bar.js

```diff
@@ -0,0 +1,54 @@
+const _ = require('lodash/fp')
+const { withRegisteredUser, withBilling, withWorkspace, performAnalysisTabSetup } = require('../utils/integration-helpers')
+const {
+  click, clickable, getAnimatedDrawer, findElement, noSpinnersAfter, delay
+} = require('../utils/integration-utils')
+
+
+const testAnalysisContextBarFn = _.flow(
+  withWorkspace,
+  withBilling,
+  withRegisteredUser
+)(async ({ page, token, testUrl, workspaceName }) => {
+  // Navigate to appropriate part of UI (the analysis tab)
+  await performAnalysisTabSetup(page, token, testUrl, workspaceName)
+
+  // Create a runtime
+  await delay(30000)
```

Comment:
If the issue is that the timeout on the next line is sometimes not long enough, please increase the default delay for that call instead of adding a delay that will increase the run time for every test run.

(I realize you are probably already trying this out, but adding a note since @kyuksel was not part of our Slack conversation)